public-api/src/main/java/org/pmiops/workbench/cdr/model/DbConcept.java

```diff
@@ -287,38 +294,50 @@ public Concept hasCounts(int hasCounts) {
     public void setSynonyms(List<String> synonyms) {
         this.synonyms = synonyms;
     }
-    public Concept synonyms(List<String> synonyms) {
+    public DbConcept synonyms(List<String> synonyms) {
         this.synonyms = synonyms;
         return this;
     }
 
     @Transient
     public List<String> getDrugBrands() { return drugBrands; }
     public void setDrugBrands(List<String> drugBrands) { this.drugBrands = drugBrands; }
-    public Concept drugBrands(List<String> drugBrands) {
+    public DbConcept drugBrands(List<String> drugBrands) {
         this.drugBrands = drugBrands;
         return this;
     }
```

Comment:
nit: can we remove these transient properties since they aren't referenced.

Reply:
drugbrands in ui are displayed from this property here https://github.com/all-of-us/data-browser/blob/master/public-ui/src/app/data-browser/views/ehr-view/ehr-view.component.ts#L411

Reply:
Yes, but the UI uses the `Concept` object not the `DbConcept` object. `DbConcept` is only used by the API code and it isn't referenced anywhere.

Reply:
Removed this but left in getter over there for the mapper to pick it up during conversion.